XCUITest/OnboardingTest.swift

```diff
@@ -39,36 +39,30 @@ class OnboardingTest: XCTestCase {
 
     // Smoketest
     func testPressingDots() throws {
-        throw XCTSkip("This test is temporarily disabled, it should be refactored when the new onboarding functionality is done")
-        let stackElement = app.otherElements["Intro.stackView"]
-        let pageIndicatorButton1 = stackElement.children(matching: .button).matching(identifier: "page indicator").element(boundBy: 0)
-        let pageIndicatorButton2 = stackElement.children(matching: .button).matching(identifier: "page indicator").element(boundBy: 1)
-        let pageIndicatorButton3 = stackElement.children(matching: .button).matching(identifier: "page indicator").element(boundBy: 2)
+        let pageIndicatorButton = app.pageIndicators.firstMatch
```

Comment:
The `pageIndicator` may work differently with just two pages. The same place is pressed to switch between pages instead of a dot for each page.